bitrise.yml

```diff
@@ -1481,19 +1481,30 @@ workflows:
         - scheme: Fennec_Enterprise
         - configuration: Fennec_Enterprise
         - project_path: "$BITRISE_PROJECT_PATH"
-    - script@1.1:
-        title: Firebase Build for Testing Physical Devices
-        inputs:
-        - content: |
-            set -euxo pipefail
-            echo "-- build-for-testing --"
-            mkdir DerivedData
-            xcodebuild -resolvePackageDependencies -onlyUsePackageVersionsFromResolvedFile
-            xcodebuild build-for-testing -project /Users/vagrant/git/Client.xcodeproj \
-                -scheme Fennec_Enterprise \
-                -sdk iphoneos \
-                -destination "generic/platform=iOS" \
-                -derivedDataPath "/Users/vagrant/git/DerivedData"
+    - deploy-to-bitrise-io@2: {}
+    # - script@1.1:
+    #     title: Firebase Build for Testing Physical Devices
```

Comment:
@jjSDET remember that we can not test this workflow in PRs. This step was working successfully in the last try so we don't need to change anything here.
The error was only (as of now) in the gcloud step. That's the only change needed.

I will uncomment and leave this step as it was and will try to land the PR with the gcloud change.